src/Converter/GhostscriptConverter.php

```diff
@@ -69,5 +69,7 @@ public function convert($file, $newVersion)
             throw new \RuntimeException("The generated file '{$tmpFile}' was not found.");
 
         $this->fs->copy($tmpFile, $file, true);
+        
+        $this->fs->delete($tmpFile);
```

Comment:
This is not working, delete is not a function of the Filesystem class of Symfony. 

https://symfony.com/doc/current/components/filesystem.html

Reply:
Updated to use `remove()`